airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/spec.json

```diff
@@ -20,6 +20,13 @@
         "description": "Required if you access Zendesk Chat from a Zendesk Support subdomain.",
         "default": ""
       },
+      "api_pagination_limit": {
+        "type": "integer",
+        "title": "Api Pagination Limit (0=No Limit)",
+        "description": "Limit up to how many pages will be queried in the API to force the writing of data",
+        "default": 0,
+        "minimum": 0
+      },
```

Comment:
@RobertoBonnet this should not be changed only after 100k requests? Adding this to users will create an overhead process here.

Reply:
My question is why not implement this https://github.com/airbytehq/airbyte/issues/12591#issuecomment-1130544537 in code instead give a complex configuration to users?
- Limit to 1000 pages the sync and finish? not the best because users will think the connector has problem and will ask why data is missing. maybe one solution will add a warning in documentation
- Implement in the connector the change of request/min after 1000 so users don't need to worry about it

@sherifnada can you give your opinion here? this is a problem that impacts users will large Zendesk accounts that got stuck because of Zendesk side